docs/content/grafana/add_grafana_dashboard.md

```diff
@@ -23,15 +23,15 @@ In this template, all you need to do is update:
 1. `metadata.name` - the dashboard name
 2. `spec.url` - the URL pointing to the location of your dashboard JSON file
 
-Pick a suitable `metadata.name`, ensure that its unique in the [`grafana/base/dashboards`][1] folder.
+Pick a suitable `metadata.name`, ensure that its unique in the [`grafana/overlays/moc/smaug/dashboards`][1] folder.
 
-Save this file under `grafana/base/dashboards/${DASHBOARD_FOLDER}/grafana-dashboard-example.yaml`.
+Save this file under `grafana/overlays/moc/smaug/dashboards/${DASHBOARD_FOLDER}/grafana-dashboard-example.yaml`.
```

Comment:
We need to add a new kustomization here.
`grafana/overlays/moc/smaug/dashboards/${DASHBOARD_FOLDER}/kustomization.yaml`

Reply:
Just to be sure, you mean replace the path in `line 28` to what you posted? Or to add a new line?